### DISCUSSION TITLE
Play Slideshow with API

### DISCUSSION THREAD

**Discussion by jaygeorge** [fancyapps/ui#266] [Q&A]
Hi, thank you for FancyBox 4, I love it.

Is it possible to 'play' a gallery slideshow using the API?

What I'd like to do is have a button next to the gallery that says 'Play slideshow'.
This would open the gallery and start the slideshow action.

Is this possible? I'm not sure, looking in the API

I can get this far:

```html
<button class="js__playFancyboxSlideshow">
    play slideshow
</button>
```

```js
Array.from(document.querySelectorAll('.js__playFancyboxSlideshow') || []).forEach(element => {
    element.onclick = function(){
        Fancybox.fromOpener('[data-fancybox="gallery"]', {
            startIndex: 0,
            // Start slideshow as an action here?
        });
    }
});
```

**Comment by fancyapps** [comment]
Hi,

So, you are looking for auto-start the slideshow? It is possible, like this:

```js
Fancybox.bind('[data-fancybox="gallery"]', {
  slideshow: {
    autoStart: true
  }
});
```

https://fancyapps.com/playground/2nY
